src/lib.rs

```diff
@@ -44,6 +44,7 @@
 
 // Experimental features we need
 #![cfg_attr(all(test, feature = "unstable"), feature(test))]
+#![cfg_attr(feature = "unstable", feature(debug_non_exhaustive))]
```

Comment:
This will be unusable for stable. We could have a feature `rust_1_53` for this but it'd change the crate policy. People usually detect Rust version using build scripts and set internal cfg.